torchqdynamics/solvers/mesolve.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
+from typing import List
+
+import torch
+
+
+@dataclass
+class MESolverOption:
+    jump_ops: List[torch.Tensor]
```

Comment:
I guess we should move this to the `mesolve` function argument too.

Reply:
Yes, in a second time, I'd time to merge this so I can fix https://github.com/PierreGuilmin/torchqdynamics/pull/5